### DISCUSSION TITLE
Unanswered questions feedback please

### DISCUSSION THREAD

**Discussion by amalmsr** [Project-MONAI/MONAI#7772] [General]
Hello MONAI Team,

First, I want to extend my gratitude for your incredible open-source work. MONAI has been instrumental in my ongoing master's studies, and I truly appreciate the effort put into this project.

I understand that there is no obligation for all questions to be answered, but I have noticed that most of my queries  (and I believe I am not alone in this) remain unaddressed. I am wondering if this could be due to the questions being too broad or perhaps unanswerable in their current form.

Could you provide guidance on whether it’s possible to receive feedback when questions are too broad or cannot be answered? This would greatly help in setting the right expectations and improving the way I ask questions in the future.

Thank you again for your amazing work and support!

Best regards!

**Comment by KumoLiu** [comment]
Hi @spes3, thanks for interest here.

To which conversation are you referring? I saw that you started this conversation a few days ago. Discussion 7735 at https://github.com/Project-MONAI/MONAI Furthermore, identifying the primary cause from the long piece of the code and even without the whole error log is not simple. The best course of action would be for you to have debugged for some time and to supply a brief code snippet that can be used to replicate the problem. This way, we could assist in identifying the core cause with ease.

Thanks.

**Reply by amalmsr** [reply]
Thank you so much! I'll make a note of your feedback and try to ask my questions in a brief way again and what debugging attempts I tried. All my questions were about my attempts with the transforms and inference using a .ts converted model/ the final .pt model of SegResNetDs (without the model ensemble), similar to the more direct app versions in MONAI Deploy. I close older discussions when I feel like I found a way around them.

**Comment by Edmon02** [comment]
I understand how frustrating it can be when questions go unanswered. A few things that might help:
	1.	Provide more details: If your question is too vague, try adding logs, screenshots, or specific examples of what you’ve tried.
	2.	Use proper formatting: Well-structured questions with clear bullet points and code blocks often get more attention.
	3.	Check similar issues: Sometimes, your question might have been answered elsewhere—searching GitHub Discussions or Stack Overflow can help.
	4.	Tag relevant people: If a maintainer or active contributor is involved in a similar topic, tagging them politely might increase visibility.

I hope this helps! If you want, I can check out your question and offer suggestions to improve its visibility. Let me know!

**Comment by manimovassagh** [comment]
Thanks for raising this! I've had similar experiences, so I want to offer a few practical tips that have helped me get better responses:

1. **Be specific with the title** – Make sure the title gives readers a clear idea of your issue (e.g., “How to use `monai.transforms` for 3D segmentation?” instead of “Help please”).
2. **Show what you've tried** – Even a short code snippet or error message shows that you've made an effort, and makes it easier for others to jump in.
3. **Link relevant docs or examples** – Mention if you've tried a particular example in MONAI's docs and what went wrong.
4. **Use tags smartly** – Tags like `help-wanted`, `question`, or related feature names (`dataloader`, `transforms`) can help get attention.
5. **Don’t be discouraged by silence** – Sometimes it's just timing. Try rephrasing your post or breaking it into smaller parts.

Happy to give feedback on a specific post if you link one. Keep asking — the community *does* want to help!
